### PR TITLE
Added documentation for Kiali

### PR DESCRIPTION
**Pull request by brian-avery**

### CODE REVIEW

**Comment by dmitri-d** [comment]
Looks good (assuming you resolve the conflicts). I quite like the style for listings (if it's new, but it's the first time I see it).

**Comment by brian-avery** [comment]
Fixes Maistra-33

File: topics/istiodocs/custom-install.adoc [hunk 1]
```diff
@@ -42,6 +42,11 @@ spec:
     prefix: distributed-tracing-tech-preview/
     version: 1.6.0
     elasticsearch_memory: 1Gi
+  kiali:
+    username: username
+    password: password
+    prefix: kiali/
+    version: v0.7
```
Comment (brian-avery) [inline topics/istiodocs/custom-install.adoc]: This needs to be changed to 0.7.1 before merging

File: topics/istiodocs/custom-install.adoc [hunk 2]
```diff
@@ -113,6 +118,33 @@ spec:
...
+|prefix
+|a prefix to apply to Kiali image names used in docker pull.
+|valid image repo
+|kiali/
+
+
+|version
+|docker tag to use with Kiali images
+|valid Kiali tag
+|0.7
```
Comment (brian-avery) [inline topics/istiodocs/custom-install.adoc]: This needs to be changed to 0.7.1 before merging.

**Pull request merged**